lib/basedriver/driver.js

```diff
@@ -259,7 +259,7 @@ class BaseDriver extends Protocol {
     this.clearNewCommandTimeout();
 
     // if we don't have this command, it must not be implemented
-    if (!this[cmd]) {
+    if (!this[cmd] && !isImageCommand(args, cmd)) {
```

Comment:
we could use `getImgElFromArgs` instead

Reply:
ah, right.
`execute` in image-element.js raise `new errors.NotYetImplementedError();` if current command doesn't exist there.
e.g.: `element.tag_name` issues `/wd/hub/session/7446e842-fca2-4e11-a9ad-6e64cbe41e93/element/appium-image-element-48bd1a97-07b4-43c0-bc3e-5e044069c763/name` and calles `getName`. But it should raise `NotYetImplementedError`. We can get the raise in https://github.com/appium/appium-base-driver/pull/245/files#diff-69c68e3397852a86397892936887db6aR292

I see..